### ISSUE TITLE
Include fragment level information in the output

### ISSUE THREAD

**Issue by ianchlee** [vdemichev/DiaNN#761]
Hi, may I ask if it is possible to output fragment level information in the output, such as raw peak area, normalised peak area, if the fragment is used to quantitate the peptides etc? It would be great if we can access the fragment level information and examine the fragment summation.

**Comment by vdemichev** [comment]
Yes, already there: Fragment.Quant.Raw column. Also see --report-lib-info.

**Comment by ianchlee** [comment]
Thanks for the information. It seems that Fragment.Quant.Raw does not sum to the fragment group quantity. Is there a way to get the FG quantity from the raw fragment output?
